### DISCUSSION TITLE
文本错误

### DISCUSSION THREAD

**Discussion by XTsat** [BeyondDimension/SteamTools#1644] [General]
![image](https://user-images.githubusercontent.com/44708609/178443397-39d33f84-1b1c-4471-a689-0f73d2b16dbb.png)

**Comment by rmbadmin** [comment]
这是有意区分的，因为在程序内显示库存管理可能会与Steam的库存概念混淆

**Comment by XTsat** [comment]
![image](https://user-images.githubusercontent.com/44708609/178461928-be156ed7-54cd-4300-a018-24bb5d8fb294.png)

英文版直接叫库了都，中文改成库管理是不是因为少个字对不齐所以现在这样

还有一个叫本地令牌，一个steam令牌，关键叫steam令牌的只有++英文版（只看中英翻译），其他都是本地令牌

这些是由于英文版的个人翻译原因是吧

还有为什么英文版的主页为啥要删几个功能介绍啊
